libs/docs/src/docs/no-version/creating-a-devfile-stack.md

```diff
@@ -9,20 +9,174 @@ various projects.
 
 ## Procedure
 
-1. Create a stack folder with a name that matches the stack name. For
-    example, `java-wildfly/`.
+1. Create a fork of the [devfile-registry repo](https://github.com/devfile/registry), clone
+your fork locally and go to its directory.
 
-2. Create version directories for storing different stack versions.
+2. Create a stack folder with a name that matches the stack name. For
+    example, `tutorial-stack/`.
+
+3. Create version directories for storing different stack versions.
     Each directory under the stack must correspond to a specific
-    version, for example, `java-wildfly/1.0.0`.
+    version, for example, `tutorial-stack/2.2.0`.
```

Comment:
this makes it seem like the version numbers are related to the `schemaVersion` when it should be the `version` number instead. maybe switch this to `1.0.3` to match the examples below?